pkg/server/http/svc_test.go

```diff
@@ -1127,6 +1127,135 @@ func TestUpdateUser(t *testing.T) {
 	}
 }
 
+// This is a bug demonstration test for endpoint '/cloud/user'
+// Link to the issue: https://github.com/owncloud/ocis-ocs/issues/52
+
+func TestGetSingleUser(t *testing.T) {
+	users := []User{
+		{
+			Enabled:     "true",
+			Username:    "rutherford",
+			ID:          "rutherford",
+			Email:       "rutherford@example.com",
+			Displayname: "Ernest RutherFord",
+		},
+		{
+			Enabled:     "true",
+			Username:    "thomson",
+			ID:          "thomson",
+			Email:       "thomson@example.com",
+			Displayname: "J. J. Thomson",
+		},
+	}
+
+	for _, ocsVersion := range ocsVersions {
+		for _, format := range formats {
+			for _, user := range users {
+				err := createUser(user)
+				if err != nil {
+					t.Fatal(err)
+				}
+			}
+
+			formatpart := getFormatString(format)
+			res, err := sendRequest(
+				"GET",
+				fmt.Sprintf("/%v/cloud/user%v", ocsVersion, formatpart),
+				"",
+				"admin:admin",
```

Comment:
This will still not work but you should use the authorization of the user you just created. The user `admin` doesn't exist.